### ISSUE TITLE
It's time to use HTTPS

### ISSUE THREAD

**Issue by BumbuKhan** [hasan-li/develolper#5]
We could get SSL certificate absolutely for free, so why don't we use it ?!

I've already purchased it and install on my own [https://bumbu.tv/](https://bumbu.tv/) (project is half cooked)
Certificate delivered by [https://letsencrypt.org/](https://letsencrypt.org/) and should be updated every 90 days (we could write a script for autoupdate)

I can install that, just provide me SSH access to your hosting

**Comment by BumbuKhan** [comment]
@hasan-li have you changed the hosting provider?

**Comment by hasan-li** [comment]
Yes. But we we can't transfer domain. It will be possible only after 60 days
